src/d3a/models/market/market_structures.py

```diff
@@ -110,15 +110,19 @@ def offer_from_JSON_string(offer_string, current_time):
     return offer
 
 
-class Bid(namedtuple('Bid', ('id', 'time', 'price', 'energy', 'buyer',
-                             'original_bid_price', 'buyer_origin', 'energy_rate'))):
+class Bid(namedtuple('Bid', (
+        'id', 'time', 'price', 'energy', 'buyer', 'original_bid_price', 'buyer_origin',
+        'energy_rate'))):
+
     def __new__(cls, id, time, price, energy, buyer, original_bid_price=None,
                 buyer_origin=None, energy_rate=None):
         if energy_rate is None:
             energy_rate = price / energy
         # overridden to give the residual field a default value
-        return super(Bid, cls).__new__(cls, str(id), time, price, energy, buyer,
-                                       original_bid_price, buyer_origin, energy_rate)
+
+        return super(Bid, cls).__new__(
+            cls, str(id), time, price, energy, buyer, original_bid_price, buyer_origin,
+            energy_rate)
```

Comment:
Curious if any specific code convention is followed here?

Reply:
The change is actually due to two separate commits that were adding some new parameter and then removing it. The style had changed in the meantime.

In terms of PEP8, the two versions are equally good: https://www.python.org/dev/peps/pep-0008/#indentation
I usually prefer using a _hanging indent_ because it saves a lot of space on the right side. If the method name is long, you might end up with something like this:
```python
# Without hanging indent you would waste a lot of space on the left, below the
# name of the method. And you might not have enough space on the right
# because of the 100 characters line limit
def a_method_with_a_very_long_name_and_accepting_lots_of_parameters(self,
                                                                    foo,
                                                                    bar,
                                                                    *args,
                                                                    **kwargs):

# With hanging indent you could use the space on the left, so you'd have:
def a_method_with_a_very_long_name_and_accepting_lots_of_parameters(
    self, foo, bar, *args, **kwargs):

```